static/panes/editor.js

```diff
@@ -267,67 +267,76 @@ Editor.prototype.initCallbacks = function () {
 };
 
 Editor.prototype.initButtons = function (state) {
-    this.fontScale = new FontScale(this.domRoot, state, this.editor);
-    this.languageBtn = this.domRoot.find('.change-language');
+    var scopedThis = this;
```

Comment:
Im not entirely sure the need for this, using bind should fix the need for the scopedThis(it was why it was introduced into jquery/js) variable in all  the cases.If there is a specific case you can't get to it to work feel free to reach out to me on slack and We can work through it.